docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -1,7 +1,7 @@
 ---
 aliases:
-- /docs/tempo/latest/server_side_metrics/span_metrics/
-- /docs/tempo/latest/metrics-generator/span_metrics/
+  - /docs/tempo/latest/server_side_metrics/span_metrics/
+  - /docs/tempo/latest/metrics-generator/span_metrics/
```

Comment:
Good catch on the indent. I had to go look it up and you're right (which means I need to update my other PR).